buildpack/build.go

```diff
@@ -208,29 +211,29 @@ func (b *Descriptor) setupEnv(pathToLayerMetadataFile map[string]LayerMetadataFi
 	return nil
 }
 
-func eachDir(dir, buildpackAPI string, fn func(path, api string) (LayerMetadataFile, error)) (map[string]LayerMetadataFile, error) {
-	files, err := ioutil.ReadDir(dir)
+func eachLayer(bpLayersDir, buildpackAPI string, fn func(path, api string) (LayerMetadataFile, error)) (map[string]LayerMetadataFile, error) {
+	files, err := ioutil.ReadDir(bpLayersDir)
 	if os.IsNotExist(err) {
 		return map[string]LayerMetadataFile{}, nil
 	} else if err != nil {
 		return map[string]LayerMetadataFile{}, err
 	}
-	pathToLayerMetadataFile := map[string]LayerMetadataFile{}
+	bpLayers := map[string]LayerMetadataFile{}
 	for _, f := range files {
-		if !f.IsDir() {
```

Comment:
This guard is probably still useful. It’s unlikely to happen but there could be a dir with that suffix.

Reply:
Please see fbbb2e66